documentation/justfile

```diff
@@ -18,6 +19,48 @@ install:
 # Sphinx #
 ##########
 
+# Create a new project with proposal and implementation plans
+init-project name *ips:
+    #!/usr/bin/env bash
+    set -eo pipefail
+    TIMESTAMP=$(date '+%Y%m%d')
+    DATE=$(date '+%Y-%m-%d')
+    PROJECTFS="{{ replace(lowercase(name), ' ', '_') }}"
+    # Create project directory
+    mkdir "projects/proposals/$PROJECTFS"
+    # Create project directory index
+    sed \
+      -e '1,3d;$d' \
+      -e 's/{Project title}/{{ name }}/g' \
+      projects/templates/index.md > "projects/proposals/$PROJECTFS/index.md"
+    # Create project proposal
+    sed \
+      -e '1,3d;$d' \
+      -e 's/{Project proposal title}/{{ name }}/g' \
+      -e "s/{YYYY-MM-DD}/$DATE/g" \
+      projects/templates/project_proposal.md > "projects/proposals/$PROJECTFS/$TIMESTAMP-project_proposal.md"
+    # Create implementation plans
+    shift
+    for IP in "$@"; do
+        just create-ip "projects/proposals/$PROJECTFS" "$IP"
+    done
+
+# Create a new implementation plan in the given directory
+create-ip directory name:
```

Comment:
Could this be `project_name` instead of `name`? Totally a nit, because I read it as `directory_name` at first :)

Reply:
Updated all arg names for clarity in 074c2b7.